projects/dotcms-js/src/lib/core/dotcms-events.service.ts

```diff
@@ -59,30 +61,32 @@ export class DotcmsEventsService {
      * regarding incoming system events. The events they will receive will be
      * based on the type of event clients register for.
      *
-     * @param clientEventType - The type of event clients will get. For example,
-     *                          "notification" will allow a client to receive the
-     *                          messages in the Notification section.
-     * @returns any The system events that a client will receive.
+     * @memberof DotcmsEventsService
```

Comment:
`param` missing

Reply:
Angulasr cried when I build this with params and return in the doc, saying is not necessary.